README.md

```diff
@@ -115,7 +115,7 @@ An incomplete list of features Argo Workflows provide:
 * Multiple executors
 * Multiple pod and workflow garbage collection strategies
 * Automatically calculated resource usage per step
-* Java/Golang/Python SDKs
+* Java and Golang SDKs
```

Comment:
I don't think this needs to be changed since there are still Python SDKs and Hera is "in progress" to being considered official (https://github.com/argoproj/argoproj/issues/286)